doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -713,8 +713,70 @@ so notification receiver can know the format used without needing processing not
 
 ### Notification templates
 
-Based on the "template" field defined as part of the subscription and in a macro replacement language
-that could be used in "payload", "url", "headers", "method" and "qs". Details TBD.
+At notification time, NGSIv2 implementations will send a notification based on the `notification.http`
+field of the associated subscription. This fields can be interpreted litearly or use a
+templating mechanism, based in macros substitution, that is described in the present subsection.
+
+The following fields inside `notification.http` in subscriptions are intepreted as templates:
+
+* `url`
+* `headers` (both header names and values)
+* `qs` (both in parameters names and values)
+* `payload`
+
+It is worth mentioning that `method` is not a template and it is always used literarly.
+
```

Comment:
Typo fixed in ffb27c6.

It think it is better to leave the sentences. Additional clarification doesn't hurt. However, I'd like to heard more opinions on this.